imexport/export.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+
+	"github.com/cozy/cozy-stack/client"
+	"github.com/cozy/cozy-stack/client/auth"
+	"github.com/cozy/cozy-stack/client/request"
+)
+
+func export(tw *tar.Writer, opts *request.Options, authReq *auth.Request) error {
+	cClient := &client.Client{
+		Domain: authReq.Domain,
+		Scheme: opts.Scheme,
+		Client: opts.Client,
+
+		AuthClient: authReq.ClientParams,
+		AuthScopes: authReq.Scopes,
+		Authorizer: opts.Authorizer,
+		UserAgent:  opts.UserAgent,
+	}
+
+	root := "/Documents"
+
+	if err := cClient.WalkByPath(root, func(path string, doc *client.DirOrFile, err error) error {
```

Comment:
[nitpick] the combo err-in-if with a lambda function make this a bit hard to read : I'd recommend either naming the lambda  or having on separate lines `err := cClient.Walk....` & `if err != nil`